go/vt/wrangler/vdiff.go

```diff
@@ -37,6 +37,7 @@ import (
 	"vitess.io/vitess/go/vt/key"
 	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/logutil"
+	vtschema "vitess.io/vitess/go/vt/schema"
```

Comment:
I don't think we should rename this import. I read it as `vschema` several times and kept wondering why IsInteralOperationTableName` was in that package.

If there's a local variable in this file preventing us from using this name, I think we should rename that variable.

Reply:
unrenamed and should be in next commit